arviz/stats/stats.py

```diff
@@ -1302,7 +1310,6 @@ def summary(
         metric_names.extend(metrics_names_)
 
     if circ_var_names:
-
```

Comment:
This was extraneous whitespace that did not seem to match the style of the surrounding code, so I removed it — hope you don't mind!